### PR TITLE
Switch2.0

### LINKED ISSUE #1266: 2.0 - Update Switch
This is a tracking issue for updating Switch

**Scope:** Switch

### PR DESCRIPTION
**Pull request by prateekbh**
fixes #1266

### CODE REVIEW

File: packages/switch/src/index.tsx [hunk 1]
```diff
@@ -17,15 +17,6 @@ export class Switch extends MaterialComponent<ISwitchProps, ISwitchState> {
...
       this.MDComponent = new MDCSwitch(this.control);
-      if (this.props.value) {
-        this.MDComponent.value = this.props.value;
-      }
-    }
-  }
-
-  public componentWillUpdate(nextProps: ISwitchProps) {
-    if (nextProps.value && this.props.value !== nextProps.value) {
-      this.MDComponent.value = nextProps.value;
```
Comment (cromefire) [inline packages/switch/src/index.tsx]: Why was this removed?

Comment (prateekbh) [reply packages/switch/src/index.tsx]: `value` setter is removed: https://github.com/material-components/material-components-web/tree/master/packages/mdc-switch#mdcswitch-properties-and-methods

Comment (cromefire) [reply packages/switch/src/index.tsx]: We should maybe think of an alternative

Comment (prateekbh) [reply packages/switch/src/index.tsx]: shouldn't `checked` be fine?

Comment (cromefire) [reply packages/switch/src/index.tsx]: yes it should
